pkgs/stdenv/darwin/default.nix

```diff
@@ -32,7 +32,7 @@ in rec {
     export NIX_ENFORCE_NO_NATIVE="''${NIX_ENFORCE_NO_NATIVE-1}"
     export NIX_IGNORE_LD_THROUGH_GCC=1
     stripAllFlags=" " # the Darwin "strip" command doesn't know "-s"
-    export MACOSX_DEPLOYMENT_TARGET=10.10
+    export MACOSX_DEPLOYMENT_TARGET=10.12
     export SDKROOT=
     export CMAKE_OSX_ARCHITECTURES=x86_64
```

Comment:
I believe this is a no-op. CMAKE_OSX_ARCHITECTURES should be in cmakeFlags, not an env var. I believe this can be removed.

Reply:
gl_cv_func_getcwd_abort_bug can be removed as well